ui/Screen/Settings.svelte

```diff
@@ -173,14 +172,14 @@
           <p class="typo-text-bold">
             Seeds help you find more projects and people on the network.
           </p>
-          <!-- TODO(julien): link to actual docs abt seeds -->
           <p
             style="color: var(--color-foreground-level-6); margin-bottom: 24px;">
             Enter seed addresses that you’d like to subscribe to here.
             <a
               style="color: var(--color-foreground-level-5);"
               class="typo-link"
-              href="https://radicle.xyz/#upstream-faq">Learn more about seeds</a>
+              href="https://docs.radicle.xyz/docs/using-radicle/running-a-seed-node">Learn
```

Comment:
I wonder if "Learn more about seeds" should not link to either [this](https://docs.radicle.xyz/docs/getting-started/getting-started#adding-a-seed-node-to-upstream) or [this](https://docs.radicle.xyz/docs/using-radicle/troubleshooting#seed-node)? These are more about what seeds are than how to spin up your own.

Reply:
yeah, but this one specifically I had it link there bc the two places i want to link it are going to be updated by outstanding prs. I could include a broken  link for now then fix later?

Reply:
@MeBrei @rudolfs - what do you think?

Reply:
If the outstanding PRs are likely to be merged soon, I would wait for them. Otherwise we could keep the link above for now.

Reply:
I'm going to try to merge the glossary PR now, so we can link to that. Will ping here if it takes longer than expected.

Reply:
Hey all - here's the link to use https://docs.radicle.xyz/docs/understanding-radicle/glossary#seed. It'll be broken till I push one change (changing `seed node` to `seed). But I'll have that fixed in a jif!